core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.ts

```diff
@@ -92,6 +92,11 @@ export class TopicEditorTabComponent implements OnInit, OnDestroy {
   uncategorizedEditOptionsIndex: number;
   subtopicEditOptionsAreShown: number;
   skillOptionDialogueBox: boolean = true;
+  MAX_CHARS_IN_TOPIC_NAME: number;
```

Comment:
Where is this being used?

Reply:
All these variables are defined in **constraints.ts.**

Reply:
Can you please provide me with the path to constraints.ts file, or add a snap shot of it? Also can you add a comment above the list of variables that are defined in constraint.ts that these variables are defined in constraints.ts file?

Reply:
Path to constraints.ts file = asserts/constraints.ts. Adding SS of place where these variables have been defined.
<img width="1680" alt="Screenshot 2023-04-29 at 1 17 39 AM" src="https://user-images.githubusercontent.com/61676272/235239796-31451ecc-50ea-462b-8098-ed60f7259676.png">